<?xml version="1.0" encoding="utf-8"?><testsuite errors="0" failures="2" name="pytest" skips="0" tests="7" time="13.155"><testcase classname="api.tests.test_fadn_pipeline" file="api/tests/test_fadn_pipeline.py" line="31" name="test_fadn" time="6.312883138656616"><failure message="assert None is not None">client_factory = &lt;function client_factory.&lt;locals&gt;.provider at 0xffffb6cfe9e0&gt;
api_test_resources = (&apos;1&apos;, &apos;user@api.demeter.org&apos;, [&apos;offline_access&apos;, &apos;access&apos;, &apos;uma_authorization&apos;], &apos;2&apos;, &apos;admin@api.demeter.org&apos;, [&apos;offline_access&apos;, &apos;access&apos;, &apos;admin&apos;, &apos;uma_authorization&apos;])

    def test_fadn(client_factory, api_test_resources):
        def test_fadn_fetch(client_factory, api_test_resources):
            (
                user_identifier,
                user_username,
                user_roles_data,
                admin_identifier,
                admin_username,
                admin_roles_data,
            ) = api_test_resources
            api_client = client_factory(user=admin_username, realm_access=admin_roles_data)
            data = {
                &quot;description&quot;: get_random_string(),
                &quot;stage&quot;: &quot;fetch&quot;,
                &quot;graph_uri&quot;: &quot;&quot;,
                &quot;graph_per_dump&quot;: False,
                &quot;reload_graph&quot;: False,
                &quot;url_input&quot;: TestFadn.URL_INPUT,
                &quot;config&quot;: &quot;&quot;,
            }
            fadn_tests_success(api_client, data)
    
        def test_fadn_preprocess(client_factory, api_test_resources):
            (
                user_identifier,
                user_username,
                user_roles_data,
                admin_identifier,
                admin_username,
                admin_roles_data,
            ) = api_test_resources
            api_client = client_factory(user=admin_username, realm_access=admin_roles_data)
            data = {
                &quot;description&quot;: get_random_string(),
                &quot;stage&quot;: &quot;preprocess&quot;,
                &quot;graph_uri&quot;: &quot;&quot;,
                &quot;graph_per_dump&quot;: False,
                &quot;reload_graph&quot;: False,
                &quot;url_input&quot;: TestFadn.URL_INPUT,
                &quot;config&quot;: &quot;&quot;,
            }
            fadn_tests_success(api_client, data)
    
&gt;       test_fadn_fetch(client_factory, api_test_resources)

api/tests/test_fadn_pipeline.py:75: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
api/tests/test_fadn_pipeline.py:52: in test_fadn_fetch
    fadn_tests_success(api_client, data)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

api_client = RequestsWrapper(&apos;http&apos;, None, &apos;&apos;)
data = {&apos;config&apos;: &apos;&apos;, &apos;description&apos;: &apos;w3Hx4fZm905Q&apos;, &apos;graph_per_dump&apos;: False, &apos;graph_uri&apos;: &apos;&apos;, ...}

    def fadn_tests_success(api_client, data):
        # post lpis data
        api_client.post(f&quot;/api/{TestFadn.PIPELINE}/&quot;, form_data=data)
        assert api_client.last.status == 201
        # post action execute
        out = api_client.last.json
        fadn_identifier = out[&quot;identifier&quot;].value
        api_client.post(f&quot;/api/{TestFadn.PIPELINE}/{fadn_identifier}/execute/&quot;)
        assert api_client.last.status == 202
        # TODO: DEM-61 - Check results of the job, not only status
    
        # checking if the action execute worked
        api_client.get(f&quot;/api/{TestFadn.PIPELINE}/{fadn_identifier}/&quot;)
        out = api_client.last.json
        fadn_output_url = out[&quot;output_url&quot;].value
        print(f&quot;xdxd={fadn_output_url}&quot;)
&gt;       assert fadn_output_url is not None
E       assert None is not None

api/tests/test_fadn_pipeline.py:29: AssertionError</failure><system-out>Operations to perform:
  Synchronize unmigrated apps: common, corsheaders, django_elasticsearch_dsl, django_extensions, django_filters, drf_yasg, messages, rest_framework, staticfiles, watchman
  Apply all migrations: admin, api, auth, contenttypes, sessions
Synchronizing apps without migrations:
  Creating tables...
    Running deferred SQL...
Running migrations:
  Applying contenttypes.0001_initial... OK
  Applying contenttypes.0002_remove_content_type_name... OK
  Applying auth.0001_initial... OK
  Applying auth.0002_alter_permission_name_max_length... OK
  Applying auth.0003_alter_user_email_max_length... OK
  Applying auth.0004_alter_user_username_opts... OK
  Applying auth.0005_alter_user_last_login_null... OK
  Applying auth.0006_require_contenttypes_0002... OK
  Applying auth.0007_alter_validators_add_error_messages... OK
  Applying auth.0008_alter_user_username_max_length... OK
  Applying auth.0009_alter_user_last_name_max_length... OK
  Applying auth.0010_alter_group_name_max_length... OK
  Applying auth.0011_update_proxy_permissions... OK
  Applying auth.0012_alter_user_first_name_max_length... OK
  Applying api.0001_initial... OK
  Applying admin.0001_initial... OK
  Applying admin.0002_logentry_remove_auto_add... OK
  Applying admin.0003_logentry_add_action_flag_choices... OK
  Applying api.0002_auto_20230120_2207... OK
  Applying api.0003_auto_20230207_1234... OK
  Applying sessions.0001_initial... OK
xdxd=None
</system-out><system-err>Creating test database for alias &apos;default&apos; (&apos;test_main-db&apos;)...
[2023-03-01 10:11:32,953] pipelines_agent.celery:10 DEBUG: DJANGO_SETTINGS_MODULE=pipelines_agent.settings
[2023-03-01 10:11:32,953] pipelines_agent.celery:12 INFO: Creating pipelines celery worker application...
[2023-03-01 10:11:32,954] pipelines_agent.celery:14 DEBUG: Pipelines celery worker application created: &lt;Celery pipelines_agent at 0xffffb6596b10&gt;
[2023-03-01 10:11:32,954] pipelines_agent.celery:17 DEBUG: Tasks auto discovery on
[2023-03-01 10:11:32,954] pipelines_agent.celery:21 DEBUG: pipelines_agent_settings_source=None
[2023-03-01 10:11:32,954] pipelines_agent.celery:25 INFO: Pipelines agent config not present
[2023-03-01 10:11:32,954] pipelines_agent.celery:28 INFO: Preparing configuration for pipelines agent
[2023-03-01 10:11:32,954] pipelines_agent.celery:37 INFO: Getting pipelines agent configuration from celery_config...
[2023-03-01 10:11:32,954] pipelines_agent.celery:39 INFO: Pipelines agent configuration applied
[2023-03-01 10:11:32,999] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:33,000] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:33,000] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {&apos;identifier&apos;: &apos;2&apos;}
[2023-03-01 10:11:33,004] api.auth:160 INFO: Roles of user admin@api.demeter.org have changed
[2023-03-01 10:11:33,007] api.auth:166 DEBUG: Role offline_access created
[2023-03-01 10:11:33,009] api.auth:166 DEBUG: Role access created
[2023-03-01 10:11:33,010] api.auth:166 DEBUG: Role admin created
[2023-03-01 10:11:33,012] api.auth:166 DEBUG: Role uma_authorization created
[2023-03-01 10:11:33,131] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:33,132] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:33,132] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:33,203] api.tasks.tasks:26 INFO: time_consuming_operation_task task started
[2023-03-01 10:11:33,204] api.tasks.tasks:35 DEBUG: Job_started_on=2023-03-01 10:11:33.204008+00:00
[2023-03-01 10:11:33,205] api.tasks.tasks:38 DEBUG: Operation was submitted by user admin@api.demeter.org
[2023-03-01 10:11:33,205] api.serializers:338 DEBUG: Fadn task_action started
[2023-03-01 10:11:33,206] api.serializers:353 DEBUG: Directory /tmp/pipelines created
[2023-03-01 10:11:33,208] api.serializers:355 DEBUG: Directory /tmp/pipelines/5c1421bc-b6d8-4eb6-8c44-a3b20ed04451_101133205948 created
[2023-03-01 10:11:33,209] api.serializers:396 DEBUG: command=cp -r /home/pipelines_user/src/cfg/config_template.yaml /home/pipelines_user/src/cfg/config.yaml;. /home/pipelines_user/pipelines/bin/activate;cd /home/pipelines_user/src; python3 main.py fadn --stage=fetch  --url_input=https://box.psnc.pl/f/27e3fee316/?raw=1 --output=/tmp/5c1421bc-b6d8-4eb6-8c44-a3b20ed04451_101133205948/results/;
[2023-03-01 10:11:33,209] api.serializers:708 DEBUG: callback_address=http://django:9001/api/fadn/5c1421bc-b6d8-4eb6-8c44-a3b20ed04451/
[2023-03-01 10:11:33,214] pipelines_agent.pipelines_tasks:24 DEBUG: Pipeline task started
[2023-03-01 10:11:33,214] pipelines_agent.pipelines_tasks:28 DEBUG: command_to_execute=cp -r /home/pipelines_user/src/cfg/config_template.yaml /home/pipelines_user/src/cfg/config.yaml;. /home/pipelines_user/pipelines/bin/activate;cd /home/pipelines_user/src; python3 main.py fadn --stage=fetch  --url_input=https://box.psnc.pl/f/27e3fee316/?raw=1 --output=/tmp/5c1421bc-b6d8-4eb6-8c44-a3b20ed04451_101133205948/results/;
[2023-03-01 10:11:33,214] pipelines_agent.pipelines_tasks:30 DEBUG: callback_address=http://django:9001/api/fadn/5c1421bc-b6d8-4eb6-8c44-a3b20ed04451/
[2023-03-01 10:11:33,214] pipelines_agent.pipelines_tasks:32 DEBUG: pipeline_id_time=5c1421bc-b6d8-4eb6-8c44-a3b20ed04451_101133205948
[2023-03-01 10:11:33,214] pipelines_agent.pipelines_tasks:34 DEBUG: nonce=gAAAAABj_yTVU6NJYMTMJ3SurMp-sX2Z-Kn_6Eu71-Ymzpdm5Vt7ODg1eRA3DAHdeXcULLNem1LeFUzpngU2uk7ZpAvYariGx3eAlsAr_SDxI2od78nhFJscvbrjStfJtPLtwn1wyUGECF8MiZ7EbBMhE8yYBE9ns4LTy9cE4yMM5-czTdNQS1TcamsP4fWEIPo-Sujg2hMfoDyjrNZyesju-52G5iYuGuqUBAceJVvnfy4MsldnBDStwmTcmxC4i-0LnpKZA4j7
[2023-03-01 10:11:33,214] pipelines_agent.pipelines_tasks:51 DEBUG: Executing command
Traceback (most recent call last):
  File &quot;main.py&quot;, line 7, in &lt;module&gt;
    from click_option_group import optgroup, MutuallyExclusiveOptionGroup, RequiredMutuallyExclusiveOptionGroup
ModuleNotFoundError: No module named &apos;click_option_group&apos;
[2023-03-01 10:11:33,272] pipelines_agent.pipelines_tasks:53 DEBUG: Execute ended
[2023-03-01 10:11:33,273] pipelines_agent.pipelines_tasks:60 DEBUG: Directory /tmp/5c1421bc-b6d8-4eb6-8c44-a3b20ed04451_101133205948/results created
[2023-03-01 10:11:33,273] pipelines_agent.pipelines_tasks:77 DEBUG: {&apos;file&apos;: &lt;File: /tmp/5c1421bc-b6d8-4eb6-8c44-a3b20ed04451_101133205948/results.zip&gt;}
[2023-03-01 10:11:33,289] pipelines_agent.pipelines_tasks:86 ERROR: Sending callback failed
[2023-03-01 10:11:33,289] pipelines_agent.pipelines_tasks:87 ERROR: HTTPConnectionPool(host=&apos;django&apos;, port=9001): Max retries exceeded with url: /api/fadn/5c1421bc-b6d8-4eb6-8c44-a3b20ed04451/ (Caused by NewConnectionError(&apos;&lt;urllib3.connection.HTTPConnection object at 0xffffb71254d0&gt;: Failed to establish a new connection: [Errno 111] Connection refused&apos;))
[2023-03-01 10:11:33,289] pipelines_agent.pipelines_tasks:88 DEBUG: Pipeline task ending
[2023-03-01 10:11:33,293] celery.app.trace:131 INFO: Task pipelines_agent.pipelines_tasks.pipeline_task[26347825-f596-4e3c-a3f9-d055802ea1a3] succeeded in 0.07838870800333098s: None
[2023-03-01 10:11:33,293] api.serializers:718 DEBUG: Pipeline task executed
[2023-03-01 10:11:33,294] api.serializers:724 DEBUG: stdout: None
[2023-03-01 10:11:33,295] api.serializers:727 DEBUG: iteration=1
[2023-03-01 10:11:34,297] api.serializers:727 DEBUG: iteration=2
[2023-03-01 10:11:35,300] api.serializers:727 DEBUG: iteration=3
[2023-03-01 10:11:36,304] api.serializers:727 DEBUG: iteration=4
[2023-03-01 10:11:37,306] api.serializers:727 DEBUG: iteration=5
[2023-03-01 10:11:38,307] api.serializers:732 DEBUG: Output_url: None
[2023-03-01 10:11:38,307] api.serializers:736 DEBUG: Ending of checking fadn pipeline
[2023-03-01 10:11:38,309] api.tasks.tasks:76 DEBUG: job=740bbefe-5e35-4d46-9eb4-19275cabd9d1
[2023-03-01 10:11:38,310] api.tasks.tasks:87 DEBUG: finished_on=2023-03-01 10:11:38.309950+00:00
[2023-03-01 10:11:38,312] api.tasks.tasks:90 INFO: time_consuming_operation_task task ended
[2023-03-01 10:11:38,312] celery.app.trace:131 INFO: Task api.tasks.tasks.time_consuming_operation_task[740bbefe-5e35-4d46-9eb4-19275cabd9d1] succeeded in 5.108847501978744s: None
[2023-03-01 10:11:38,318] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:38,318] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:38,318] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
</system-err></testcase><testcase classname="api.tests.test_lpis_pipeline" file="api/tests/test_lpis_pipeline.py" line="31" name="test_lpis" time="5.2328362464904785"><failure message="assert None is not None">client_factory = &lt;function client_factory.&lt;locals&gt;.provider at 0xffffb703bf80&gt;
api_test_resources = (&apos;1&apos;, &apos;user@api.demeter.org&apos;, [&apos;offline_access&apos;, &apos;access&apos;, &apos;uma_authorization&apos;], &apos;2&apos;, &apos;admin@api.demeter.org&apos;, [&apos;offline_access&apos;, &apos;access&apos;, &apos;admin&apos;, &apos;uma_authorization&apos;])

    def test_lpis(client_factory, api_test_resources):
        def test_lpis_fetch(client_factory, api_test_resources):
            (
                user_identifier,
                user_username,
                user_roles_data,
                admin_identifier,
                admin_username,
                admin_roles_data,
            ) = api_test_resources
            api_client = client_factory(user=admin_username, realm_access=admin_roles_data)
            data = {
                &quot;description&quot;: get_random_string(),
                &quot;stage&quot;: &quot;fetch&quot;,
                &quot;graph_uri&quot;: &quot;&quot;,
                &quot;graph_per_dump&quot;: False,
                &quot;reload_graph&quot;: False,
                &quot;url_input&quot;: TestLpis.URL_INPUT,
                &quot;country&quot;: &quot;&quot;,
                &quot;other&quot;: &quot;&quot;,
                &quot;config&quot;: &quot;&quot;,
            }
            lpis_tests_success(api_client, data)
    
        def test_lpis_mapping(client_factory, api_test_resources):
            (
                user_identifier,
                user_username,
                user_roles_data,
                admin_identifier,
                admin_username,
                admin_roles_data,
            ) = api_test_resources
            api_client = client_factory(user=admin_username, realm_access=admin_roles_data)
            data = {
                &quot;description&quot;: get_random_string(),
                &quot;stage&quot;: &quot;mapping&quot;,
                &quot;graph_uri&quot;: &quot;&quot;,
                &quot;graph_per_dump&quot;: False,
                &quot;reload_graph&quot;: False,
                &quot;url_input&quot;: TestLpis.URL_INPUT,
                &quot;country&quot;: &quot;spain&quot;,
                &quot;other&quot;: &quot;&quot;,
                &quot;config&quot;: &quot;&quot;,
            }
            lpis_tests_success(api_client, data)
    
        def test_lpis_transform(client_factory, api_test_resources):
            (
                user_identifier,
                user_username,
                user_roles_data,
                admin_identifier,
                admin_username,
                admin_roles_data,
            ) = api_test_resources
            api_client = client_factory(user=admin_username, realm_access=admin_roles_data)
            data = {
                &quot;description&quot;: get_random_string(),
                &quot;stage&quot;: &quot;transform&quot;,
                &quot;graph_uri&quot;: &quot;&quot;,
                &quot;graph_per_dump&quot;: False,
                &quot;reload_graph&quot;: False,
                &quot;url_input&quot;: TestLpis.URL_INPUT,
                &quot;country&quot;: &quot;spain&quot;,
                &quot;other&quot;: &quot;&quot;,
                &quot;config&quot;: &quot;&quot;,
            }
            lpis_tests_success(api_client, data)
    
        def test_lpis_postprocess(client_factory, api_test_resources):
            (
                user_identifier,
                user_username,
                user_roles_data,
                admin_identifier,
                admin_username,
                admin_roles_data,
            ) = api_test_resources
            api_client = client_factory(user=admin_username, realm_access=admin_roles_data)
            data = {
                &quot;description&quot;: get_random_string(),
                &quot;stage&quot;: &quot;postprocess&quot;,
                &quot;graph_uri&quot;: &quot;&quot;,
                &quot;graph_per_dump&quot;: False,
                &quot;reload_graph&quot;: False,
                &quot;url_input&quot;: TestLpis.URL_INPUT,
                &quot;country&quot;: &quot;spain&quot;,
                &quot;other&quot;: &quot;&quot;,
                &quot;config&quot;: &quot;&quot;,
            }
            lpis_tests_success(api_client, data)
    
&gt;       test_lpis_fetch(client_factory, api_test_resources)

api/tests/test_lpis_pipeline.py:125: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
api/tests/test_lpis_pipeline.py:54: in test_lpis_fetch
    lpis_tests_success(api_client, data)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

api_client = RequestsWrapper(&apos;http&apos;, None, &apos;&apos;)
data = {&apos;config&apos;: &apos;&apos;, &apos;country&apos;: &apos;&apos;, &apos;description&apos;: &apos;Pe7ouoFgFVFm&apos;, &apos;graph_per_dump&apos;: False, ...}

    def lpis_tests_success(api_client, data):
        # post lpis data
        api_client.post(f&quot;/api/{TestLpis.PIPELINE}/&quot;, form_data=data)
        assert api_client.last.status == 201
    
        # post action execute
        out = api_client.last.json
        lpis_identifier = out[&quot;identifier&quot;].value
        api_client.post(f&quot;/api/{TestLpis.PIPELINE}/{lpis_identifier}/execute/&quot;)
        assert api_client.last.status == 202
        # TODO: DEM-61 - Check results of the job, not only status
    
        # checking if the action execute worked
        api_client.get(f&quot;/api/{TestLpis.PIPELINE}/{lpis_identifier}/&quot;)
        out = api_client.last.json
        lpis_output_url = out[&quot;output_url&quot;].value
&gt;       assert lpis_output_url is not None
E       assert None is not None

api/tests/test_lpis_pipeline.py:29: AssertionError</failure><system-err>[2023-03-01 10:11:38,506] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:38,506] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:38,506] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {&apos;identifier&apos;: &apos;2&apos;}
[2023-03-01 10:11:38,510] api.auth:160 INFO: Roles of user admin@api.demeter.org have changed
[2023-03-01 10:11:38,512] api.auth:166 DEBUG: Role offline_access created
[2023-03-01 10:11:38,515] api.auth:166 DEBUG: Role access created
[2023-03-01 10:11:38,517] api.auth:166 DEBUG: Role admin created
[2023-03-01 10:11:38,519] api.auth:166 DEBUG: Role uma_authorization created
[2023-03-01 10:11:38,526] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:38,527] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:38,527] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:38,535] api.tasks.tasks:26 INFO: time_consuming_operation_task task started
[2023-03-01 10:11:38,536] api.tasks.tasks:35 DEBUG: Job_started_on=2023-03-01 10:11:38.536062+00:00
[2023-03-01 10:11:38,536] api.tasks.tasks:38 DEBUG: Operation was submitted by user admin@api.demeter.org
[2023-03-01 10:11:38,537] api.serializers:186 DEBUG: Lpis task_action started
[2023-03-01 10:11:38,538] api.serializers:199 DEBUG: Removing old results
[2023-03-01 10:11:38,539] api.serializers:206 DEBUG: Directory /tmp/pipelines created
[2023-03-01 10:11:38,540] api.serializers:208 DEBUG: Directory /tmp/pipelines/924f2b43-7809-4d42-bacd-2296705c37b0_101138537576 created
[2023-03-01 10:11:38,540] api.serializers:266 DEBUG: command= cp -r /home/pipelines_user/src/cfg/config_template.yaml /home/pipelines_user/src/cfg/config.yaml;. /home/pipelines_user/pipelines/bin/activate;cd /home/pipelines_user/src; python3 main.py lpis --stage=fetch   --url_input=https://box.psnc.pl/f/5b80101bc9/?raw=1 --output=/tmp/924f2b43-7809-4d42-bacd-2296705c37b0_101138537576/results/;
[2023-03-01 10:11:38,540] api.serializers:708 DEBUG: callback_address=http://django:9001/api/lpis/924f2b43-7809-4d42-bacd-2296705c37b0/
[2023-03-01 10:11:38,540] pipelines_agent.pipelines_tasks:24 DEBUG: Pipeline task started
[2023-03-01 10:11:38,540] pipelines_agent.pipelines_tasks:28 DEBUG: command_to_execute= cp -r /home/pipelines_user/src/cfg/config_template.yaml /home/pipelines_user/src/cfg/config.yaml;. /home/pipelines_user/pipelines/bin/activate;cd /home/pipelines_user/src; python3 main.py lpis --stage=fetch   --url_input=https://box.psnc.pl/f/5b80101bc9/?raw=1 --output=/tmp/924f2b43-7809-4d42-bacd-2296705c37b0_101138537576/results/;
[2023-03-01 10:11:38,540] pipelines_agent.pipelines_tasks:30 DEBUG: callback_address=http://django:9001/api/lpis/924f2b43-7809-4d42-bacd-2296705c37b0/
[2023-03-01 10:11:38,540] pipelines_agent.pipelines_tasks:32 DEBUG: pipeline_id_time=924f2b43-7809-4d42-bacd-2296705c37b0_101138537576
[2023-03-01 10:11:38,540] pipelines_agent.pipelines_tasks:34 DEBUG: nonce=gAAAAABj_yTaxRk6NpYVEgWNLjPFyZgGmqR25dv-Lt9JblZxxdBTKwCYiHDEHjwysi1TNhsMU7h7q4AXX5BdYjF_lUWt3Nk_HUeYLpgIMOArC7ht65pVQzH-0ZZ5Vi7T6vasg7TJEeNOP1FR9Kq2SGQFPFIbN8-4Xxs6kKYYs0xfzES1ArDUXPlmC8pYA-aNCTPLGHzk-AeDLU4nMJuDUXDBEBtn5FyH3ypnsiYMTsPgK77mOdflpJPKdNIt_kIKEbX3fghn5B0f
[2023-03-01 10:11:38,540] pipelines_agent.pipelines_tasks:51 DEBUG: Executing command
Traceback (most recent call last):
  File &quot;main.py&quot;, line 7, in &lt;module&gt;
    from click_option_group import optgroup, MutuallyExclusiveOptionGroup, RequiredMutuallyExclusiveOptionGroup
ModuleNotFoundError: No module named &apos;click_option_group&apos;
[2023-03-01 10:11:38,602] pipelines_agent.pipelines_tasks:53 DEBUG: Execute ended
[2023-03-01 10:11:38,602] pipelines_agent.pipelines_tasks:60 DEBUG: Directory /tmp/924f2b43-7809-4d42-bacd-2296705c37b0_101138537576/results created
[2023-03-01 10:11:38,602] pipelines_agent.pipelines_tasks:77 DEBUG: {&apos;file&apos;: &lt;File: /tmp/924f2b43-7809-4d42-bacd-2296705c37b0_101138537576/results.zip&gt;}
[2023-03-01 10:11:38,605] pipelines_agent.pipelines_tasks:86 ERROR: Sending callback failed
[2023-03-01 10:11:38,605] pipelines_agent.pipelines_tasks:87 ERROR: HTTPConnectionPool(host=&apos;django&apos;, port=9001): Max retries exceeded with url: /api/lpis/924f2b43-7809-4d42-bacd-2296705c37b0/ (Caused by NewConnectionError(&apos;&lt;urllib3.connection.HTTPConnection object at 0xffffb6db3950&gt;: Failed to establish a new connection: [Errno 111] Connection refused&apos;))
[2023-03-01 10:11:38,605] pipelines_agent.pipelines_tasks:88 DEBUG: Pipeline task ending
[2023-03-01 10:11:38,606] celery.app.trace:131 INFO: Task pipelines_agent.pipelines_tasks.pipeline_task[cb6ca43a-07b0-40b5-ade2-c39a7e8dd917] succeeded in 0.06535845901817083s: None
[2023-03-01 10:11:38,606] api.serializers:718 DEBUG: Pipeline task executed
[2023-03-01 10:11:38,607] api.serializers:724 DEBUG: stdout: None
[2023-03-01 10:11:38,608] api.serializers:727 DEBUG: iteration=1
[2023-03-01 10:11:39,610] api.serializers:727 DEBUG: iteration=2
[2023-03-01 10:11:40,613] api.serializers:727 DEBUG: iteration=3
[2023-03-01 10:11:41,615] api.serializers:727 DEBUG: iteration=4
[2023-03-01 10:11:42,619] api.serializers:727 DEBUG: iteration=5
[2023-03-01 10:11:43,623] api.serializers:732 DEBUG: Output_url: None
[2023-03-01 10:11:43,624] api.serializers:736 DEBUG: Ending of checking lpis pipeline
[2023-03-01 10:11:43,633] api.tasks.tasks:76 DEBUG: job=5caad756-1158-4ec7-9f03-e2bc52cb151c
[2023-03-01 10:11:43,634] api.tasks.tasks:87 DEBUG: finished_on=2023-03-01 10:11:43.634571+00:00
[2023-03-01 10:11:43,645] api.tasks.tasks:90 INFO: time_consuming_operation_task task ended
[2023-03-01 10:11:43,646] celery.app.trace:131 INFO: Task api.tasks.tasks.time_consuming_operation_task[5caad756-1158-4ec7-9f03-e2bc52cb151c] succeeded in 5.108504960022401s: None
[2023-03-01 10:11:43,659] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:43,661] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:43,661] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
</system-err></testcase><testcase classname="api.tests.tests" file="api/tests/tests.py" line="12" name="test_description" time="0.10825324058532715"><system-err>[2023-03-01 10:11:43,770] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:43,771] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:43,771] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {&apos;identifier&apos;: &apos;2&apos;}
[2023-03-01 10:11:43,773] api.auth:160 INFO: Roles of user admin@api.demeter.org have changed
[2023-03-01 10:11:43,775] api.auth:166 DEBUG: Role offline_access created
[2023-03-01 10:11:43,777] api.auth:166 DEBUG: Role access created
[2023-03-01 10:11:43,780] api.auth:166 DEBUG: Role admin created
[2023-03-01 10:11:43,782] api.auth:166 DEBUG: Role uma_authorization created
[2023-03-01 10:11:43,789] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:43,790] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:43,790] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:43,799] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:43,800] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:43,800] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
</system-err></testcase><testcase classname="api.tests.tests" file="api/tests/tests.py" line="66" name="test_app" time="0.159104585647583"><system-err>[2023-03-01 10:11:43,895] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:43,895] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:43,896] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {&apos;identifier&apos;: &apos;2&apos;}
[2023-03-01 10:11:43,900] api.auth:160 INFO: Roles of user admin@api.demeter.org have changed
[2023-03-01 10:11:43,902] api.auth:166 DEBUG: Role offline_access created
[2023-03-01 10:11:43,904] api.auth:166 DEBUG: Role access created
[2023-03-01 10:11:43,906] api.auth:166 DEBUG: Role admin created
[2023-03-01 10:11:43,907] api.auth:166 DEBUG: Role uma_authorization created
[2023-03-01 10:11:43,922] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:43,923] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:43,923] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:43,935] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:43,936] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:43,936] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:43,943] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:43,943] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:43,943] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:43,950] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:43,951] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:43,951] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:43,957] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:43,957] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:43,957] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
</system-err></testcase><testcase classname="api.tests.tests" file="api/tests/tests.py" line="103" name="test_users_list" time="0.19921588897705078"><system-err>[2023-03-01 10:11:44,055] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,056] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,056] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {&apos;identifier&apos;: &apos;2&apos;}
[2023-03-01 10:11:44,060] api.auth:160 INFO: Roles of user admin@api.demeter.org have changed
[2023-03-01 10:11:44,061] api.auth:166 DEBUG: Role offline_access created
[2023-03-01 10:11:44,063] api.auth:166 DEBUG: Role access created
[2023-03-01 10:11:44,066] api.auth:166 DEBUG: Role admin created
[2023-03-01 10:11:44,068] api.auth:166 DEBUG: Role uma_authorization created
[2023-03-01 10:11:44,077] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,078] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,078] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:44,095] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,095] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,095] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:44,106] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,107] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,108] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:44,118] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,119] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,119] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:44,135] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,136] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,136] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:44,145] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,146] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,146] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:44,156] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,156] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,156] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
</system-err></testcase><testcase classname="api.tests.tests" file="api/tests/tests.py" line="243" name="test_users" time="0.6974468231201172"><system-err>[2023-03-01 10:11:44,213] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,214] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,214] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {&apos;identifier&apos;: &apos;2&apos;}
[2023-03-01 10:11:44,218] api.auth:160 INFO: Roles of user admin@api.demeter.org have changed
[2023-03-01 10:11:44,220] api.auth:166 DEBUG: Role offline_access created
[2023-03-01 10:11:44,221] api.auth:166 DEBUG: Role access created
[2023-03-01 10:11:44,224] api.auth:166 DEBUG: Role admin created
[2023-03-01 10:11:44,227] api.auth:166 DEBUG: Role uma_authorization created
[2023-03-01 10:11:44,235] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,236] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,236] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:44,245] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,245] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,245] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:44,259] api.auth:81 DEBUG: Token for user user@api.demeter.org is valid
[2023-03-01 10:11:44,259] api.auth:100 DEBUG: User user@api.demeter.org already exists
[2023-03-01 10:11:44,259] api.auth:106 DEBUG: Changed attributes of user user@api.demeter.org: {&apos;identifier&apos;: &apos;1&apos;}
[2023-03-01 10:11:44,261] api.auth:160 INFO: Roles of user user@api.demeter.org have changed
[2023-03-01 10:11:44,262] api.auth:168 DEBUG: Role offline_access already exists
[2023-03-01 10:11:44,263] api.auth:168 DEBUG: Role access already exists
[2023-03-01 10:11:44,266] api.auth:168 DEBUG: Role uma_authorization already exists
[2023-03-01 10:11:44,274] api.auth:81 DEBUG: Token for user user@api.demeter.org is valid
[2023-03-01 10:11:44,275] api.auth:100 DEBUG: User user@api.demeter.org already exists
[2023-03-01 10:11:44,275] api.auth:106 DEBUG: Changed attributes of user user@api.demeter.org: {}
[2023-03-01 10:11:44,298] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,299] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,299] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:44,311] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,312] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,312] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:44,322] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,323] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,323] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:44,334] api.auth:81 DEBUG: Token for user user@api.demeter.org is valid
[2023-03-01 10:11:44,335] api.auth:100 DEBUG: User user@api.demeter.org already exists
[2023-03-01 10:11:44,335] api.auth:106 DEBUG: Changed attributes of user user@api.demeter.org: {}
[2023-03-01 10:11:44,346] api.auth:81 DEBUG: Token for user user@api.demeter.org is valid
[2023-03-01 10:11:44,347] api.auth:100 DEBUG: User user@api.demeter.org already exists
[2023-03-01 10:11:44,347] api.auth:106 DEBUG: Changed attributes of user user@api.demeter.org: {}
[2023-03-01 10:11:44,359] api.auth:81 DEBUG: Token for user admin@api.demeter.org is valid
[2023-03-01 10:11:44,359] api.auth:100 DEBUG: User admin@api.demeter.org already exists
[2023-03-01 10:11:44,359] api.auth:106 DEBUG: Changed attributes of user admin@api.demeter.org: {}
[2023-03-01 10:11:44,365] django.request:230 ERROR: Not Implemented: /api/users/2/refresh/
[2023-03-01 10:11:44,371] api.auth:81 DEBUG: Token for user user@api.demeter.org is valid
[2023-03-01 10:11:44,372] api.auth:100 DEBUG: User user@api.demeter.org already exists
[2023-03-01 10:11:44,372] api.auth:106 DEBUG: Changed attributes of user user@api.demeter.org: {}
[2023-03-01 10:11:44,381] api.auth:81 DEBUG: Token for user user@api.demeter.org is valid
[2023-03-01 10:11:44,382] api.auth:100 DEBUG: User user@api.demeter.org already exists
[2023-03-01 10:11:44,382] api.auth:106 DEBUG: Changed attributes of user user@api.demeter.org: {}
</system-err></testcase><testcase classname="common.cache.tests.TestCache" file="common/cache/tests.py" line="6" name="test_pymemcache_client" time="0.1395883560180664"><system-err>Destroying test database for alias &apos;default&apos; (&apos;test_main-db&apos;)...
</system-err></testcase></testsuite>